.cirrus.yml

```diff
@@ -268,17 +280,15 @@ task:
         - git clone https://github.com/flutter/tests.git bin/cache/pkg/tests
         - dart --enable-asserts dev/customer_testing/run_tests.dart --skip-on-fetch-failure --skip-template bin/cache/pkg/tests/registry/*.test
 
-    - name: firebase_test_lab_tests-linux # linux-only
-      environment:
-        # Empirically, this shard runs in 20-25 minutes with just one CPU and 4G of RAM, as of
-        # October 2019. It does not seem to be sensitive to the number of CPUs or amount of RAM;
-        # doubling CPUs had no effect (mere seconds under 20 minutes), increasing RAM to 24G left it
-        # on the high end of the 20-25 minute range. (This makes sense, as it's just driving the
-        # Firebase test lab remotely.) Less than 4G of RAM made it go OOM.
-        CLOUDSDK_CORE_DISABLE_PROMPTS: 1
-        GCLOUD_FIREBASE_TESTLAB_KEY: ENCRYPTED[1c140257edc48f5578fa5a0e5038b84c8e53270c405efa5a8e35ea303a4e0d135853989f448f72136206de854d17fbec]
-      script:
-        - ./dev/bots/firebase_testlab.sh
+    # firebase_test_lab_tests are linux-only
+    - name: firebase_test_lab_tests-0-linux
+      <<: *FIREBASE_SHARD_TEMPLATE
+
+    - name: firebase_test_lab_tests-1-linux
+      <<: *FIREBASE_SHARD_TEMPLATE
+
+    - name: firebase_test_lab_tests-2-linux
+      <<: *FIREBASE_SHARD_TEMPLATE
```

Comment:
I'd really much rather we just duplicate all the text. i think the burden on people who look at this file without knowing Yaml's more obscure features is going to be much greater than the burden on people who have to deal with duplicated text.

Reply:
> just use the same style the rest of the file

Would you prefer https://github.com/flutter/flutter/pull/50306 be reverted?  I would guess so based on https://github.com/flutter/flutter/pull/50306#issuecomment-583177860.

Reply:
Well, that does end up saving a lot of duplication, huh...

Ok, so long as we have just the one mechanism for doing this, I'm good. The syntax you use here is exactly the same as in #50306?

Reply:
Yup!
https://github.com/flutter/flutter/blob/master/.cirrus.yml#L573-L583
etc